src/Components/Artwork/CollectorSignalsContext.tsx

```diff
@@ -0,0 +1,25 @@
+import { createContext, useContext } from "react"
+
+interface CollectorSignalsContextProps {
+  showCollectorSignalBadge?: boolean
+}
+
+const CollectorSignalsContext = createContext<CollectorSignalsContextProps>({
+  showCollectorSignalBadge: false,
```

Comment:
Specifically for this line, which can be one of the signals, like "Limited-time offer", "Increased demand", etc. I call it badge but open to naming suggestions.

Reply:
Now I think about it, we probably need a flag for each applicable signals, like limited-time offer, increased demand, curator's pick, etc, to determine which signals to consider under a context (specific rail or grid). If that's aligned with folks' thought, I'll make the change.

![Table](https://github.com/user-attachments/assets/8ec95367-09f2-496f-9e53-acf4321be906)

Reply:
I've been thinking about [using the `ArtworkGridContext` for this](https://github.com/artsy/force/blob/10903dd8f595094e4560c2c453a978a4901b7d35/src/Components/ArtworkGrid/ArtworkGridContext.tsx#L7-L22) - Which I think should be the place we put these new context values rather than a new collector signals-specific context provider.

1 reason is that it already has a prop that does almost the same thing: `hideLotLabel`. It also has some other props that seem to be auction specific: `saveToDefaultList` and `isAuctionArtwork`. Taken together I think we should
- Use this context (which is not specific to artwork grids but [can wrap a collection like a rail as well](https://github.com/artsy/force/blob/246ba39f72ff05ccbeed171478e3f0afe3ce1631/src/Apps/Auction/Components/AuctionWorksByFollowedArtistsRail.tsx#L22-L42)) around all list views
- Make the existing auction-related props more intuitive
- **To your point**, allow some kind of prop to ignore specific signals in a given list.

Reply:
Multiple contexts seem ok for this _assuming_ the feature will grow in complexity. Otherwise, for the time being, might be worth sharing this configuration and renaming the context. 

Another thought is: if this configuration option is going to be shared at a given collection level, surfacing this config from MP might be worthwhile, vs in code. And then that can be applied to Eigen.